### PR TITLE
use cdn.scratch.ly in staging

### PR DESCRIPTION
**Pull request by benjiwheeler**
Use `cdn.scratch.ly`, not `scratch.ly`, for static assets in staging.

Needs to be merged at the same time as matching changes in scratchr2: https://github.com/LLK/scratchr2/pull/5201

### CODE REVIEW

File: src/views/credits/credits.jsx [hunk 1]
```diff
@@ -35,7 +36,7 @@ const Credits = () => (
                                 <a href={`https://scratch.mit.edu/users/${person.userName}/`}>
                                     <Avatar
                                         alt=""
-                                        src={`https://cdn.scratch.mit.edu/get_image/user/${person.userId || 'default'}_80x80.png`}
+                                        src={thumbnailUrl(person.userId, 80)}
```
[the remark this answers is not in the document]
Comment (benjiwheeler) [reply src/views/credits/credits.jsx]: Done

**Comment by benjiwheeler** [comment]
Result after deploy to staging:

screen shot 2018-11-20 at 9 34 19 am

**Comment by BryceLTaylor** [comment]
@benjiwheeler Could you add testing steps to this?

**Comment by benjiwheeler** [comment]
Resolves https://github.com/LLK/scratch-www/issues/2215

**Pull request merged**